README.md

```diff
@@ -69,18 +69,19 @@ and flags are:
   -gomaxprocs int
     	Setting for runtime.GOMAXPROCS, <1 doesn't change the default
   -grpc
-    	Use GRPC (health check) for load testing
-  -grpc-port int
-    	grpc port (default 8079)
+       Use GRPC (health check) for load testing
```

Comment:
I'm lost on how to correct this whitespace.

Reply:
![screen shot 2018-02-09 at 4 36 15 pm](https://user-images.githubusercontent.com/3664595/36056215-61526632-0db7-11e8-8a4c-e3bdda94ca7d.png)

tab vs spaces - just keep the exact same mix as the existing lines

Reply:
fixing it for you